datanode/gateway/graphql/schema.graphql

```diff
@@ -2279,6 +2286,63 @@ type Order {
   liquidityProvision: LiquidityProvision
 }
 
+"An order update in Vega, if active it will be on the order book for the market"
+type OrderUpdate {
+  "Hash of the order data"
+  id: ID!
+
+  "The worst price the order will trade at (e.g. buy for price or less, sell for price or more) (uint64)"
+  price: String!
+
+  "The timeInForce of order (determines how and if it executes, and whether it persists on the book)"
+  timeInForce: OrderTimeInForce!
+
+  "Whether the order is to buy or sell"
+  side: Side!
+
+  "The market the order is trading on (probably stored internally as a hash of the market details)"
+  marketId: ID!
+
+  "Total number of contracts that may be bought or sold (immutable) (uint64)"
+  size: String!
+
+  "Number of contracts remaining of the total that have not yet been bought or sold (uint64)"
+  remaining: String!
+
+  "The party who place the order (probably stored internally as the party's public key)"
+  partyId: ID!
+
+  "RFC3339Nano formatted date and time for when the order was created (timestamp)"
+  createdAt: String!
+
+  "Expiration time of this order (ISO-8601 RFC3339+Nano formatted date)"
+  expiresAt: String
+
+  "The status of an order, for example 'Active'"
+  status: OrderStatus!
+
+  "The external reference (if available) for the order"
+  reference: String!
+
+  "Type the order type (defaults to PARTY)"
```

Comment:
Something's not quite right about this description

Reply:
indeed looks weird